packages/event-bus/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
what do you think about adding `"printWidth": 120,`?

Reply:
Looks good to me. Let's update the other projects too then.